json_serialization_writer.go

```diff
@@ -27,6 +27,8 @@ func (w *JsonSerializationWriter) writeRawValue(value string) {
 	w.writer = append(w.writer, value)
 }
 func (w *JsonSerializationWriter) writeStringValue(value string) {
+	value = strings.ReplaceAll(strings.ReplaceAll(value, "\t", ""), "\r", "")
```

Comment:
```suggestion
	value = strings.ReplaceAll(strings.ReplaceAll(value, "\t", "\\t"), "\r", "\\r")
```

(and move this below the next one so it doesn't end up escaping the \ multiple times)